### ISSUE TITLE
vugu/vugu/js: missing `Eqaul()` method from go1.16.

### ISSUE THREAD

**Issue by leafbebop** [vugu/vugu#189]
Go 1.16 added an Equal() method.
While I am not sure how to handle it to make it work with older Go versions, I think it is warranted to be added in `vugu/vugu/js`.

**Comment by bradleypeabody** [comment]
Yup, good point.  This should definitely be done.  Possibly there are other functions missing too that have been added, it's worth a look.

I'll leave this open for now until I get to it.  PRs welcome.

**Comment by owenwaller** [comment]
I am going to close this. If we look to support the last 2 Go versions (currently 1.20 and 1.20) then we will be supporting the latest versions of `syscall/js`. I don't think we should be adding functions to `vugu/vugu/js` just to support older Go versions.
